SparseGrids/Benchmarks/benchFFT.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef _TASMANIAN_BENCHMARK_FFT_HPP
+#define _TASMANIAN_BENCHMARK_FFT_HPP
+
+#include "benchCommon.hpp"
+
+bool benchmark_fft(std::deque<std::string> &args){
+    if (args.size() < 5) return false;
+
+    // report the test parameters to reference later
+    cout << "fourier";
+    for(auto &s : args) cout << " " << s;
+    cout << endl;
+
+    auto grid_family = GridFamily::fourier;
+
+    int num_dimensions, num_depth, iteratons, num_jumps;
+    TypeDepth dtype;
+
+    auto riter = readEntries(args.begin(), num_dimensions, num_depth, dtype, iteratons, num_jumps);
```

Comment:
* read order as one of the entries
* also read the rule, even if it is redundant to say `fourier` twice